users.go

```diff
@@ -838,7 +838,6 @@ type PersonalAccessToken struct {
 	Revoked   bool       `json:"revoked"`
 	CreatedAt *time.Time `json:"created_at"`
 	Scopes    []string   `json:"scopes"`
-	UserID    string     `json:"user_id"`
```

Comment:
Should this be an `int` instead of a `string`? I think that should solve the issue right?